Doc/faq/programming.rst

```diff
@@ -1164,6 +1164,27 @@ This converts the list into a set, thereby removing duplicates, and then back
 into a list.
 
 
+How do you remove multiple items from a list
+--------------------------------------------
+
+As with removing duplicates, iterating in reverse is one solution::
+
+    for i in range(len(mylist)-1, -1, -1):
+        if remove(mylist[i]):
+            del mylist[i]
+
+
+For a long enough list with enough deletions, moving keep items to
```

Comment:
I think this would read better if "keep" were in quotes, or maybe if it said "items to keep", or maybe "kept" (although I realize the goal is to use the same name as the predicate function). Or maybe a predicate of "should_keep" would be better.

Reply:
I went with 'kept' (without quotes) and 'should_keep()'